cmd/tailscale/cli/debug.go

```diff
@@ -287,6 +288,7 @@ func debugCmd() *ffcli.Command {
 					fs.StringVar(&ts2021Args.host, "host", "controlplane.tailscale.com", "hostname of control plane")
 					fs.IntVar(&ts2021Args.version, "version", int(tailcfg.CurrentCapabilityVersion), "protocol version")
 					fs.BoolVar(&ts2021Args.verbose, "verbose", false, "be extra verbose")
+					fs.StringVar(&ts2021Args.aceHost, "ace", "", "if non-empty, add this ACE server IP/hostname as a candidate path")
```

Comment:
Since this is a debug command, should we force the connection through ACE? Otherwise, we may not be testing what we think we’re testing.

Reply:
I meant that if support asks someone to run `tailscale debug ts2021 --ace=ace.bradfi.tz`, it looks like controlhttp.Dialer will not try to dial the host directly or resolve using DNS? If so, then the help should be:
```suggestion
					fs.StringVar(&ts2021Args.aceHost, "ace", "", "if non-empty, use this ACE server IP/hostname to dial the control plane")
```

Reply:
changed help text